### ISSUE TITLE
open now?

### ISSUE THREAD

**Issue by chaserx** [openlexington/esl#14]
I'm really not sure how ESL classes work if it's a structured thing or more like tutoring. We have dates and times available, could highlight which ones are open now.

**Comment by JunHuang01** [comment]
Agreed, this is probably not a Dayton problem, where all their classes are year round.  Whereas for some of our classes are only in service in a specific period of time.
